packages/frontend/core/src/providers/modal-provider.tsx

```diff
@@ -216,7 +216,9 @@ export const SignOutConfirmModal = () => {
 
   const onConfirm = useAsyncCallback(async () => {
     setOpen(false);
-    await signOutCloud();
+    await signOutCloud({
+      redirect: false,
+    });
```

Comment:
is this related?

Reply:
There is jump logic below this line, If redirected here, the code below will not be run.

Reply:
The previously added workspaceListStatus.loading judgment is to solve the page not found error of jumping to cloud workspace after sign out.